src/app/components/DateFormatter/__tests__/index.test.tsx

```diff
@@ -5,8 +5,8 @@ import { buildDatetime } from '../../../../../internals/getBuildData'
 
 describe('intlDateTimeFormat', () => {
   it('should format date to human readable format', () => {
-    expect(intlDateTimeFormat(1656932251000)).toEqual('7/4/2022, 10:57:31 AM')
-    expect(intlDateTimeFormat(new Date(Date.UTC(2022, 6, 27, 8, 33, 10)))).toEqual('7/27/2022, 8:33:10 AM')
+    expect(intlDateTimeFormat(1656932251000)).toEqual('7/4/2022, 10:57:31 AM')
+    expect(intlDateTimeFormat(new Date(Date.UTC(2022, 6, 27, 8, 33, 10)))).toEqual('7/27/2022, 8:33:10 AM')
```

Comment:
Perhaps replace with `\u202F` to make it more obvious

Reply:
done.